src/test/java/hw2/Ex2.java

```diff
@@ -6,13 +6,10 @@
 import org.openqa.selenium.support.ui.WebDriverWait;
 import org.testng.annotations.Test;
 import org.testng.asserts.SoftAssert;
-
+import java.util.Arrays;
 import java.util.List;
 
-import static org.testng.Assert.assertTrue;
-
-
-public class Ex2 extends firstAndFinalCommonSteps {
+public class Ex2 extends FirstAndFinalCommonSteps {
```

Comment:
The pull request should not contain any other changes except changes according to the HW3